projects/dsp-ui/src/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../../../../assets/style/viewer";
+
+// styles for CKEDITOR
+
+::ng-deep .ck-content code {
```

Comment:
@kilchenmann @mdelez for the sake of encapsulation, do you thin k this is fine? We did the same for the image viewer.

Reply:
For the moment it's fine. But keep in mind that `ng-deep` is deprecated since two (or three) Angular versions. There's no replacement for now. So, you can keep it. And one day we will have to update it.